src/tainting/Dataflow_tainting.ml

```diff
@@ -1253,24 +1276,14 @@ let check_function_signature env fun_exp args args_taints =
                        in
                        Some
                          (`Return
-                           (arg_taints
+                           (arg_taints |> Taints.of_list
```

Comment:
If there are no taints (or we fail to compute them) do we really want to return an empty set of taints here?

Reply:
I think it doesn't matter, because the `Return []` case below would behave the same as if the item hadn't existed in the folded list in the first place, but I've changed it to early return `None` on the empty list anyways.